CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-### 0.6.1 (Next)
+### 0.7.0 (Next)
```

Comment:
The version in https://github.com/ashkan18/graphlient/blob/master/lib/graphlient/version.rb should be incremented with this.